src/lib/y2partitioner/widgets/rescan_devices_button.rb

```diff
@@ -28,13 +29,21 @@ def handle
     private
 
       def continue?
-        Yast::Popup.YesNo(
+        Yast::Popup.YesNo(rescan_message)
+      end
+
+      def rescan_message
+        if Yast::Mode.installation
           # TRANSLATORS
-          format(
-            _("Current changes will be discarted, including changes done by the proposal.\n" \
-              "Do you want to continue?")
-          )
-        )
+          _("Re-scanning the storage devices will invalidate all the configuration options\n"\
+            "set in the installer regarding storage, with no possibility to withdraw.\n\n" \
+            "That includes the result and settings of the guided setup as well as the manual\n"\
+            "changes performed in the expert partitioner.")
```

Comment:
79 characters per line. I would say that's too much for 25x80. Just consider the popup border and so on.